.github/workflows/pr.yaml

```
@@ -51,3 +52,14 @@ jobs:
     with:
       build_type: pull-request
+  docs-preview:
+    name: Build Docs Preview
+    needs:
+      - test-conda
+      - test-wheels
+    secrets: inherit
+    uses: ./.github/workflows/build-docs.yml
+    with:
+      is_release: false
+      pr_number: ${{ replace(github.ref_name, 'pull-request/', '') }}
```

actionlint (1.7.11)

[error] undefined function "replace". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)